Godeps/_workspace/src/github.com/kr/pty/doc.go

```diff
@@ -0,0 +1,16 @@
+// Package pty provides functions for working with Unix terminals.
```

Comment:
Actually, `kr/pty` is used server side to create a pty/tty pair to interface with the container. The code that sets up a tty for the client comes from `docker/docker` and it looks like it supports Windows.